pkgs/applications/editors/emacs/generic.nix

```diff
@@ -1,7 +1,7 @@
-{ pname ? "emacs"
+{ pname
 , version
 , versionModifier ? ""
-, name ? "emacs-${version}${versionModifier}"
+, name ? "${pname}-${version}${versionModifier}"
```

Comment:
This is unused?

Reply:
Dunno. I have found no evidence of usage for it on Nixpkgs.
Do you believe it can be removed?

Reply:
We don't use this in the emacs-overlay, if it helps.